Build/GameSettings.cc

```diff
@@ -186,7 +186,7 @@ void InitGameOptions()
 	gGameOptions.fSciFi						 = TRUE;
 	gGameOptions.ubDifficultyLevel = DIF_LEVEL_EASY;
 	//gGameOptions.fTurnTimeLimit		 = FALSE;
-	gGameOptions.fIronManMode			 = FALSE;
+	gGameOptions.ubGameSaveMode			 = DIF_CAN_SAVE;
```

Comment:
So, the difference between DIF and GIO:
DIF is a GameSettings option and is universally used everywhere the game mode is checked.
GIO contains all values DIF contains, but in addition has the value NUM_SAVE_OPTIONS. This value is used in some loops used in the GameInitOptionsScreen.
This is true for all other options (NUM_GUN_OPTIONS, NUM_GAME_STYLES, NUM_DIFF_SETTINGS) and enhances readability a bit in these instances. I would leave it that way.